src/api/services/types.ts

```diff
@@ -12,7 +10,8 @@ export enum Grade {
 export type MemberIdentity = {
   id: string; // lowercase
   name: string;
-  cohort: Cohort;
+  cohort: number;
```

Comment:
> 기존 cohort에 의존중인 컴포넌트가 많아, 우선 cohort 데이터를 유지하고 cohorts 데이터를 추가하였습니다.
추후 다른 변경사항들을 고려해서 점진적으로 수정해야 할 것으로 보입니다!

```suggestion
  currentCohort: number;
```
이런 변경은 추후 PR에서 반영하신다는 말씀이실까요?
될 수 있으면 이 PR에서 반영되면 좋을 것 같긴 했습니다! 변경사항이 꽤 많군요?